aliphysics-ci.markdown

```diff
@@ -10,6 +10,7 @@ If a release validation fails, one can try out the release candidate by sourcing
 nightly environment from CVMFS:
 
     source /cvmfs/alice-nightlies.cern.ch/etc/login.sh
+    export PATH=/cvmfs/alice-nightlies.cern.ch/bin:$PATH
```

Comment:
So instead of running those two commands, one should simply specify the full path to `alienv`:

    /cvmfs/alice-nightlies.cern.ch/bin/alienv enter blahblah

Does this work for you?